AccessibilitySnapshot/SnapshotTesting/Classes/SnapshotTesting+Accessibility.swift

```diff
@@ -0,0 +1,195 @@
+import SnapshotTesting
+import AccessibilitySnapshot
+
+extension Snapshotting where Value == UIView, Format == UIImage {
+
+    /// Snapshots the current view with colored overlays of each accessibility element it contains, as well as an
+    /// approximation of the description that VoiceOver will read for each element.
+    ///
+    /// Matches functionality of CashApp's `SnapshotVerifyAccessibility` function.
```

Comment:
We should get rid of all of the "Matches CashApp's x function" since this _is_ the library now.